Gordon360/Controllers/NewsController.cs

```diff
@@ -110,6 +111,19 @@ public ActionResult<IEnumerable<StudentNewsCategoryViewModel>> GetCategories()
             return Ok(result);
         }
 
+        [HttpGet]
+        [Route("unapproved")]
+        [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.NEWS_APPROVAL)]
+        public ActionResult<IEnumerable<StudentNewsViewModel>> GetNewsUnapproved()
+        {
+            var result = _newsService.GetNewsUnapproved();
+            if (result == null)
+            {
+                return NotFound();
+            }
+            return Ok(result);
```

Comment:
I know that you're just copying this pattern from elsewhere, but generally we prefer to return NotFound only when the request is for an invalid resource. In a case like this, the request is valid but the data is empty, so we don't need to return not found, we can just return an empty list.
```suggestion
            var result = _newsService.GetNewsUnapproved();
            return Ok(result);
```